client/verta/verta/_tracking/entity.py

```diff
@@ -164,7 +164,7 @@ def _create_proto(cls, conn, *args, **kwargs):
     def _create_proto_internal(cls, conn, ctx, name, desc=None, tags=None, attrs=None, date_created=None, **kwargs):  # recommended params
         raise NotImplementedError
 
-    def log_code(self, exec_path=None, repo_url=None, commit_hash=None, overwrite=False, autocapture=True):
+    def log_code(self, exec_path=None, repo_url=None, commit_hash=None, overwrite=False, is_dirty=None, autocapture=True):
```

Comment:
I'm not sure about this positioning, but I think it's the best option right now.

1. It's after after `overwrite` to maintain API backwards compatibility
2. It's before `autocapture` because I want "informational" parameters like `is_dirty` to be tend left, and "behavioral" parameters like `autocapture` to tend right.